Command/CreateJobCommand.php

```diff
@@ -118,7 +117,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $worker->getJobManager()->save($job);
 
-        return 0;
+        return $this::FAILURE;
```

Comment:
Shouldn't this be $this::SUCCESS?

Similar to L72 in PruneCommand.php? Or maybe at least the result of save() if save has a return type?

Reply:
Not entirely sure why I used failure there, but now I'm keen to re-review all of my changes. Will get a change pushed